shell/platform/windows/accessibility_bridge_delegate_win32.cc

```diff
@@ -39,6 +39,7 @@ void AccessibilityBridgeDelegateWin32::OnAccessibilityEvent(
       break;
     case ui::AXEventGenerator::Event::FOCUS_CHANGED:
       DispatchWinAccessibilityEvent(win_delegate, EVENT_OBJECT_FOCUS);
+      SetFocus(win_delegate);
```

Comment:
why not just win_delegate->SetFocus()

Reply:
Purely so I can capture the call in the test :(